tests/integration/pilot/common/apps.go

```diff
@@ -216,6 +219,16 @@ func SetupApps(t resource.Context, i istio.Instance, apps *EchoDeployments) erro
 			AutoRegisterVM:    true,
 			Subsets:           []echo.SubsetConfig{{}},
 			WorkloadOnlyPorts: common.WorkloadPorts,
+		}).
+		WithConfig(echo.Config{
+			Service:   DeltaSvc,
+			Namespace: apps.Namespace,
+			Ports:     common.EchoPorts,
+			Subsets: []echo.SubsetConfig{{
+				Annotations: echo.NewAnnotations().Set(echo.SidecarProxyConfig, `proxyMetadata:
+  ISTIO_META_ISTIO_DELTA_XDS: "true"`),
```

Comment:
What does `ISTIO_META_ISTIO_DELTA_XDS` do?

Reply:
I think it directs xds traffic to the delta api

Reply:
xdsType := "GRPC"
	if features.DeltaXds {
		xdsType = "DELTA_GRPC"
	}

	opts = append(opts,
		option.NodeID(cfg.ID),
		option.NodeType(cfg.ID),
		option.PilotSubjectAltName(cfg.Metadata.PilotSubjectAltName),
		option.ProxyViaAgent(cfg.Metadata.ProxyViaAgent),
		option.OutlierLogPath(cfg.Metadata.OutlierLogPath),
		option.ProvCert(cfg.Metadata.ProvCert),
		option.DiscoveryHost(discHost),
		option.XdsType(xdsType))

I donot understand how `ISTIO_META_ISTIO_DELTA_XDS` translated to xdsType

Reply:
I think you are right, should be ISTIO_DELTA_XDS

Reply:
We donot have doc about this env either

Reply:
its in https://preliminary.istio.io/latest/docs/reference/commands/pilot-agent/#envvars. its an experimental feature for now, so not any good docs for (or ever - its not a use facing functionality, just performance optimization)

Reply:
Updated and verified that it correctly uses delta now

Reply:
Thanks